tests/observability/metrics/conftest.py

```
@@ -1166,3 +1167,21 @@ def created_fake_data_volume_resource(namespace, admin_client):
 @pytest.fixture()
 def metric_cdi_import_pods_high_restart_initial_value(prometheus):
     return int(get_metrics_value(prometheus=prometheus, metrics_name=KUBEVIRT_CDI_IMPORT_PODS_HIGH_RESTART))
+
+
+@pytest.fixture()
+def daemonsets_desired_number_scheduled(admin_client):
+    return {ds.name: str(ds.instance.status.desiredNumberScheduled) for ds in [ds for ds in DaemonSet.get()]}
```

Pylint (3.3.7)

[refactor] Unnecessary use of a comprehension, use list(DaemonSet.get()) instead.

(R1721)